src/screens/OtherProfile/index.js

```diff
@@ -606,14 +608,7 @@ const OtherProfile = () => {
     });
   };
 
-  const __handleOnEndReached = () => {
-    if (!isLastPage) {
-      getOtherFeeds(postOffset);
-    }
-  };
-
   const handleRefresh = () => {
-    setIsLastPage(false);
     getOtherFeeds(0);
   };
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `handleRefresh` function, you're calling `getOtherFeeds(0)`. This seems to be resetting the feed to the initial state. However, there's no indication that the loading states (`loading`, `initLoading`) are being reset. If these states aren't managed properly, it could lead to unexpected UI behavior. Make sure to handle these states correctly when refreshing.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed